### PR TITLE
fix(delighted): replace myAxios utility with handleHttpRequest utility

### PR DESCRIPTION
**Pull request by mihir-4116**
## What are the changes introduced in this PR?
- This pr replaces myAxios utility with handleHttpRequest utility for delighted destination

## What is the related Linear task?

Resolves INT-1909

## Please explain the objectives of your changes below

Put down any required details on the broader aspect of your changes. If there are any dependent changes, **mandatorily** mention them here

### Any changes to existing capabilities/behaviour, mention the reason & what are the changes ?

N/A

### Any new dependencies introduced with this change?

N/A

### Any new generic utility introduced or modified. Please explain the changes.

N/A

### Any technical or performance related pointers to consider with the change?

N/A

@coderabbitai review

<hr>

### Developer checklist

- [ ] My code follows the style guidelines of this project

- [ ] **No breaking changes are being introduced.**

- [ ] All related docs linked with the PR?

- [ ] All changes manually tested?

- [ ] Any documentation changes needed with this change?

- [ ] Is the PR limited to 10 file changes?

- [ ] Is the PR limited to one linear task?

- [ ] Are relevant unit and component test-cases added?

### Reviewer checklist

- [ ] Is the type of change in the PR title appropriate as per the changes?

- [ ] Verified that there are no credentials or confidential data exposed with the changes.

### CODE REVIEW

**Comment by devops-github-rudderstack** [comment]
Test report for this run is available at: https://test-integrations-dev.s3.amazonaws.com/integrations-test-reports/rudder-transformer/3237/test-report.html

**Comment by sanpj2292** [comment]
Can we add some test-cases ?

**Review by sanpj2292 — DISMISSED** [review]

File: src/v0/destinations/delighted/util.js [hunk 1]
```diff
@@ -50,53 +74,47 @@ const userValidity = async (channel, Config, userId) => {
...
+  if (
+    processedResponse &&
+    processedResponse.response &&
+    processedResponse.status === 200 &&
+    Array.isArray(processedResponse.response)
+  ) {
+    return processedResponse.response.length > 0;
+  }
+
+  throw new NetworkInstrumentationError('Invalid response');
```
Comment (aashishmalik) [inline src/v0/destinations/delighted/util.js:117]: Should we throw instrumentation error here?

Comment (sanpj2292) [reply src/v0/destinations/delighted/util.js]: Can you check my changes once and confirm if they are looking good ?

**Review by sanpj2292 — APPROVED** [review]

**Review by aashishmalik — APPROVED** [review]

**Pull request merged**